Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -80,136 +80,161 @@ public function execute()
+                    
+                    if (!$singleStoreMode) {
                         $stores = $this->_storeManager->getStores();
+                        
                         foreach ($stores as $key => $store) {
                             $_stores[$store->getCode()] = $store->getId();
                         }
                     }
+                    
                     $rootNodeId = $store->getRootCategoryId();
                     $rootCat = $this->_categoryFactory->create();
                     $cat_info = $rootCat->load($rootNodeId);
 
-                        $alreadyexist = [];
-                        $categorycollection = $this->_categoryFactory->create()->getCollection()->addAttributeToSelect('name');
-                        $exist_categories_name = [];
-                        $exist_categories_path = [];
-                        $exist_categories_pathname = [];
-                    foreach ($categorycollection as $key => $value) {
-                        $exist_categories_name[$value->getId()] = $value->getName();
-                        $exist_categories_path[$value->getId()] = $value->getPath();
-                        $checkcat = $this->_categoryFactory->create();
-                        $categoryobj = $checkcat->load($value->getId());
-                        $parentcatnames = [];
-                        $parentid = '';
-                        foreach ($this->getparentCategories($categoryobj) as $key => $parentcate) {
-                            $parentcatnames[] = $parentcate->getName();
-                            $parentid = $parentcate->getId();
+                    $alreadyExist = [];
+                    $categoryCollection = $this->_categoryFactory->create()->getCollection()->addAttributeToSelect('name');
+                    $existingCategoriesPath = [];
+                    $existingCategoriesPathName = [];
+                        
+                    foreach ($categoryCollection as $key => $value) {
+                        $existingCategories[$value->getId()] = $value->getName();
+                        $existingCategoriesPath[$value->getId()] = $value->getPath();
+                        $checkCat = $this->_categoryFactory->create();
+                        $categoryObj = $checkCat->load($value->getId());
+                        $parentCatNames = [];
+                        $parentId = '';
+                        
+                        foreach ($this->_getParentCategories($categoryObj) as $key => $parentCate) {
+                            $parentCatNames[] = $parentCate->getName();
+                            $parentId = $parentCate->getId();
                         }
-                        $parent_cat = implode('/', $parentcatnames);
-                        if ($parent_cat && $parentid) {
-                            $exist_categories_pathname[$parentid] = $parent_cat;
+                        
+                        $parent_cat = implode('/', $parentCatNames);
```

Comment:
Please use consistent property naming conventions. You have used camelCase and now under_scores - please pick one and stick to it.